CMP1903_A1_2324/Game.cs

```diff
@@ -3,19 +3,59 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.Threading;
 
 namespace CMP1903_A1_2324
 {
     internal class Game
     {
-        /*
-         * The Game class should create three die objects, roll them, sum and report the total of the three dice rolls.
-         *
-         * EXTRA: For extra requirements (these aren't required though), the dice rolls could be managed so that the
-         * rolls could be continous, and the totals and other statistics could be summarised for example.
-         */
+        // Instantiate 3 objects of the die
+        Die die1 = new Die();
+        Die die2 = new Die();
+        Die die3 = new Die();
+        int rollNum1;
+        int rollNum2;
+        int rollNum3;
+        int[] rolls;
 
-        //Methods
+        public void ExecuteGame()
+        {
+            // Rolling each dice, then putting them into an array to be iterated through to write to the console
+            // Time in-between each roll allows for the seed/timestamp for the random number to be different, ensuring a pseudo-random number every time
+            rollNum1 = die1.Roll();
+            Thread.Sleep(1);
```

Comment:
Good method to get around random being linked to timestamp

Reply:
Very much appreciated :)